providers/src/airflow/providers/__init__.py

```diff
@@ -16,7 +16,9 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from __future__ import annotations
+# We do not use "from __future__ import annotations" here because it is not supported
+# by Pycharm when we want to make sure all imports in airflow work from namespace packages
+# Adding it automatically is excluded in pyproject.toml via I002 ruff rule exclusion
```

Comment:
`from __future__ import annotations` isn't needed here anyway since there is no code with type hints

Reply:
Yep.